images/spz/spz/views.py

```diff
@@ -190,8 +193,21 @@ def registrations_import():
             return None
 
     flash('Datei konnte nicht gelesen werden', 'negative')
-    return None
+    return dict(form=form)
+
+@login_required
+@templated('internal/registrations.html')
+def registrations_verify():
+    form = forms.TagVerifyForm()
+
+    if form.validate_on_submit():
+        num = form.get_tag()
+        if models.verify_tag(num):
+            flash('Matrikelnummer "' + num + '" ist in bereits in der Datenbank', 'success')
```

Comment:
Which template do you mean? I think everything in this file works with flash messages. If I change that, it would cause inconstistency.